interface/patient_file/summary/immunizations.php

```diff
@@ -56,6 +58,7 @@
             ordering_provider = ?";
         $sqlBindArray = array(
             trim($_GET['id']),
+            UuidRegistry::isValidUUID($_GET['uuid']) ? UuidRegistry::uuidToBytes($_GET['uuid']) : null,
```

Comment:
good idea on the isValidUUID function. I'd make it clear whether it is checking for a string or binary uuid.